scripts/foglamp

```diff
@@ -19,6 +19,19 @@
 set -e
 #set -x
 
+# RedHat specific
+platform=$([ `cat /etc/os-release | grep -c -i -E "centos|red hat"` -eq 0 ] && echo debian || echo redhat)
```

Comment:
Please update the check here also, if possible.

Reply:
switch to use get_platform